### ISSUE TITLE
[Docs Fix] Update links to sample generated outputs from built-in conf

### ISSUE THREAD

**Issue by bradleyhrc** [GoogleCloudPlatform/ops-agent#1835]
**Describe the bug**
The linked samples in the [Test config generation manually](https://github.com/GoogleCloudPlatform/ops-agent/blob/master/dev-docs/dev.md#test-config-generation-manually) section of the dev-docs are broken. They should be updated to what I believe are these new paths:
*   Sample generated [golden fluent bit main conf](https://github.com/GoogleCloudPlatform/ops-agent/blob/master/confgenerator/testdata/goldens/builtin/golden/linux/fluent_bit_main.conf) at `$CONFIG_OUT/fluent_bit_main.conf`.
*   Sample generated [golden fluent bit parser conf](https://github.com/GoogleCloudPlatform/ops-agent/blob/master/confgenerator/testdata/goldens/builtin/golden/linux/fluent_bit_parser.conf) at `$CONFIG_OUT/fluent_bit_parser.conf`.
*   Sample generated [golden otel conf](https://github.com/GoogleCloudPlatform/ops-agent/blob/master/confgenerator/testdata/goldens/builtin/golden/linux/otel.yaml) at `$CONFIG_OUT/otel.conf`.

**Comment by braydonk** [comment]
I think the right output for otel is `otel.yaml`. The other two look right. Do they not match what happens when running the instructions?

**Comment by bradleyhrc** [comment]
Yeah you're right it's `otel.yaml` -- I forgot to update the text. The link in it points to `otel.yaml`. The results match, just that the respective links are currently broken in the current dev doc.

**Comment by braydonk** [comment]
Oh right I understand what the issue is now. Yeah, those links are to the old format for our `testdata` folder, which has since changed. Those updated URLs seem right to me.
